src/pageLayouts/frc/FrcHome/FrcHome.js

```diff
@@ -27,17 +27,20 @@ const FrcHome = (props) => {
         <div className="frcHome__events-container">
           <Events
             name="ILITE Summer Camps"
-            eventLink="ilite.us"
+            eventLink="outreach/SummerCamps"
             date="June, 2018"
             image="images/outreach/summercamps/stem1.jpg" />
+          
+          {/* eventlink should route to the "hosted events" page */}
+          
           <Events
             name="FTC Events"
-            eventLink="ilite.us"
+            eventLink="HOSTEDEVENTS.js"
```

Comment:
This is causing a 404 error

Reply:
This one should be throwing a 404, as we don't have a hosted events page. The comment above it does some explaining, but I just put in   eventLink="HOSTEDEVENTS.js"  so I would know what to change, once we made an events page.

Reply:
@OttCS20 Thanks for the explanation; that makes sense. However, for links that don't resolve, you can put in the expected url/file name, but you should comment out the whole block. That way the worst thing we'll end up with is a missing link instead of a broken link

Reply:
Alright! Sounds good